### ISSUE TITLE
Parameter Sizzle not working in browser

### ISSUE THREAD

**Issue by dandy** [scribbletune/scribbletune#136]
Hi,

Does that param sizzle works in browser? According to my understanding, setting sizzle value will put different volume(velocity?) to each note. But I hear same volume for all notes.

https://playcode.io/669227/

Apologies if thats not the correct channel for asking questions!
BR and Thanks

**Comment by walmik** [comment]
You re right @dandy ! We do an [early return](https://github.com/scribbletune/scribbletune/blob/master/src/clip.ts#L116) in case of browser clip. We hand over the sequence generation to Tone for browser clips. Having said that, this should be doable. I ll try to address this once I get some time. Do feel free to implement it tho! PRs are very welcome :)
